l1-contracts/src/core/Rollup.sol

```diff
@@ -18,7 +18,9 @@ import {IStaking, AttesterConfig, Exit, AttesterView, Status} from "@aztec/core/
 import {IValidatorSelection, IEmperor} from "@aztec/core/interfaces/IValidatorSelection.sol";
 import {IVerifier} from "@aztec/core/interfaces/IVerifier.sol";
 import {TempCheckpointLog, CheckpointLog} from "@aztec/core/libraries/compressed-data/CheckpointLog.sol";
```

Comment:
I wasn't able to get forge scripts to work properly with the blanket FeeLib.sol, but I am probably missing an obvious setup step